ngrinder-controller/build.gradle

```diff
@@ -2,48 +2,49 @@ description = "ngrinder-controller"
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-security', version: springboot_version)
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa', version: springboot_version)
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-cache', version: springboot_version)
+    compile (group: 'org.springframework.boot', name: 'spring-boot-loader', version: springboot_version)
+
     compile (group: "rome", name: "rome", version: "1.0")
     compile (group: "com.ibm.icu", name: "icu4j", version: "4.6")
     compile (group: "sonia.svnkit", name: "svnkit-dav", version: svnkit_version)
     compile (group: "sonia.svnkit", name: "svnkit", version: svnkit_version)
     compile (group: "javax.servlet.jsp", name: "jsp-api", version: "2.1")
     compile (group: "org.python", name: "jython-standalone", version: "2.5.3")
-    compile (group: "org.freemarker", name: "freemarker", version: "2.3.20")
     compile (group: "net.sf.ehcache", name: "ehcache-core", version: "2.5.1")
     compile (group: "com.google.guava", name: "guava", version: "12.0")
-    compile (group: "org.springframework", name: "spring-web", version: spring_version)
-    compile (group: "org.springframework", name: "spring-tx", version: spring_version)
-    compile (group: "org.springframework", name: "spring-context-support", version: spring_version)
-    compile (group: "org.springframework", name: "spring-aspects", version: spring_version)
-    compile (group: "org.springframework", name: "spring-expression", version: spring_version)
     compile (group: "org.springframework.security", name: "spring-security-web", version: spring_security_version)
     compile (group: "org.springframework.security", name: "spring-security-taglibs", version: spring_security_version)
     compile (group: "org.springframework.security", name: "spring-security-config", version: spring_security_version)
     compile (group: "org.apache.maven", name: "maven-embedder", version: "3.0.5")
     compile (group: "org.sonatype.aether", name: "aether-connector-asynchttpclient", version: "1.13.1")
-    compile (group: "org.liquibase", name: "liquibase-core", version: "2.0.5")
-    compile (group: "org.springframework", name: "spring-webmvc", version: spring_version)
-    compile (group: "org.hibernate.javax.persistence", name: "hibernate-jpa-2.0-api", version: "1.0.0.Final")
-    compile (group: "org.hibernate", name: "hibernate-entitymanager", version: "4.2.2.Final")
-    compile (group: "org.hibernate", name: "hibernate-ehcache", version: "4.2.2.Final")
+    compile (group: "org.liquibase", name: "liquibase-core", version: "3.5.3")
+    compile (group: "org.hibernate", name: "hibernate-ehcache", version: "5.0.11.Final")
     compile (group: "org.aspectj", name: "aspectjrt", version: "1.6.12")
     compile (group: "org.aspectj", name: "aspectjweaver", version: "1.6.12")
```

Comment:
Do we need above two lines?

Reply:
It seems unnecessary,  `aspectjweaver` is auto imported by starter-data-jpa and have all classes in aspectjrt